### PR TITLE
CB-11282 ensure that we have proper tracking in EDH for securityAcces…

### PR DESCRIPTION
**Pull request by akanto**
…sType. Which means that we would like to track whether customers are using CIDR or security groups to allow access their clusters

See detailed description in the commit message.

### CODE REVIEW

**Review by nadamdb — APPROVED** [review]

File: environment/src/main/java/com/sequenceiq/environment/environment/dto/SecurityAccessDto.java [hunk 1]
```diff
@@ -38,12 +49,22 @@ public static Builder builder() {
 
     public static class Builder {
 
+        private static final String TYPE_CIDR_WIDE_OPEN = "CIDR_WIDE_OPEN";
```
Comment (bergerdenes) [inline environment/src/main/java/com/sequenceiq/environment/environment/dto/SecurityAccessDto.java:52]: smells like it should have been an enum

**Review by bergerdenes — CHANGES_REQUESTED** [review]
please make the new field as an enum

File: environment/src/main/java/com/sequenceiq/environment/environment/dto/SecurityAccessDto.java (hunk 1 above)
Comment (akanto) [reply environment/src/main/java/com/sequenceiq/environment/environment/dto/SecurityAccessDto.java:52]: They are converted to String anyway, I would rather delete the static declarations

Comment (akanto) [reply environment/src/main/java/com/sequenceiq/environment/environment/dto/SecurityAccessDto.java]: done

**Review by bergerdenes — APPROVED** [review]

**Review by biharitomi — APPROVED** [review]
LGTM

**Pull request merged**